database.hh

```diff
@@ -870,7 +879,20 @@ public:
     }
 
     bool compaction_enforce_min_threshold() const {
-        return _config.compaction_enforce_min_threshold;
+        return _config.compaction_enforce_min_threshold || _is_receiving_streaming;
+    }
+
+    unsigned min_compaction_threshold() {
+        // During receiving stream, (bootstrap or replace) the less we
+        // compact the faster streaming is. There are no readers so it is
+        // fine to be less aggressive with compactions as long as we don't
+        // ignore them completely (this could create a problem for when
+        // streaming ends)
+        if (_is_receiving_streaming) {
+            return 16;
```

Comment:
Should be: std::max(schema()->min_compaction_threshold(), 16)

Reply:
indeed, will fix.